<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProtonVPN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="refreshBtn">
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusLbl">
          <property name="text">
           <string>Current Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusVal">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="addressLbl">
          <property name="text">
           <string>Current IP-Address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="addressVal">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="loadingStat">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="credentials">
          <property name="text">
           <string>Credentials</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reconnectBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reconnect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="securecoreBtn">
          <property name="text">
           <string>Secure Core Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="torBtn">
          <property name="text">
           <string>TOR Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="p2pBtn">
          <property name="text">
           <string>P2P Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fastBtn">
          <property name="text">
           <string>Fastest Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lastBtn">
          <property name="text">
           <string>Last Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="randomBtn">
          <property name="text">
           <string>Random Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="connectBtn">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuSpecial_Connect">
     <property name="title">
      <string>Special Connect</string>
     </property>
     <addaction name="actionSecure_Core"/>
     <addaction name="actionTOR"/>
     <addaction name="actionPeer_to_peer"/>
     <addaction name="actionFastest"/>
     <addaction name="actionLast"/>
     <addaction name="actionRandom"/>
    </widget>
    <addaction name="actionRefresh"/>
    <addaction name="actionCredentials"/>
    <addaction name="menuSpecial_Connect"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCredentials">
   <property name="text">
    <string>Credentials</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSecure_Core">
   <property name="text">
    <string>Secure Core</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionTOR">
   <property name="text">
    <string>TOR</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionPeer_to_peer">
   <property name="text">
    <string>Peer to peer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionFastest">
   <property name="text">
    <string>Fastest</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="text">
    <string>Last</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRandom">
   <property name="text">
    <string>Random</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
